contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,87 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
+The patch release czar works with SIGs to coordinate merging cherrypicks into the release branch,
+then cutting and publishing a new patch release.
+
+## Communicating fixes to the community
+
+The canonical source of issues in a patch release will be communicated through GitHub issues.
+
+To make it easy for the community to understand the status of each patch release, GitHub issues
+must follow a consistent format and contain basic information about the issue.
+
+GitHub issues targeted at patch releases must contain the following items:
+
+### Release label
+
+The release label contains information about which releases are impacted by the issue, and
+which release the issue is resolved in.
+
+- `release/introduced-in/1.7.0`
+- `release/resolved-in/1.7.2`
+
+### Priority label
+
+The priority label describes the urgency and severity of the issue.
+
+- `priority/critical-urgent`: the issue is severe enough that it requires an immediate patch release
+  (storage or network issues that can cause data corruption or outages for instance)
+- `priority/important-soon`: the issue should be fixed in a patch does not need to be fixed immediately
```

Comment:
To avoid ambiguity.  Implicitly defaulting the severity means we can't differentiate between "someone forgot to add a label to a critical issue" and "this can wait on this"